src/orders/components/OrderSendRefundPage/messages.ts

```diff
@@ -0,0 +1,100 @@
+import { defineMessages } from "react-intl";
+
+export const refundPageMessages = defineMessages({
+  backlink: {
+    id: "rVIlBs",
+    defaultMessage: "Order #{orderNumber}",
+    description: "page header with order number",
+  },
+  backlinkNoNumber: {
+    id: "6u4K7e",
+    defaultMessage: "Order",
+    description: "page header",
+  },
```

Comment:
I believe I saw it multiple times already

Reply:
ok, refactored it into one place